spec/spec_helper.rb

```diff
@@ -1,5 +1,6 @@
 require "bundler/setup"
 require "daru/io"
+require 'fakeredis/rspec'
 
 RSpec::Expectations.configuration.warn_about_potential_false_positives = false
```

Comment:
I wanted to have mock tests on Redis, and hence `fakeredis`. (I read somewhere that redis-server is a bit inconsistent on Travis, and Travis itself has fakeredis inside to mock the Redis server)